### ISSUE TITLE
TypeError: runtime_deserialize_legacy() missing 1 required positional argument: 'size'

### ISSUE THREAD

**Issue by MoonBunnyZZZ** [Nomow/tensorrt5#1]
Hello!
I change the offical python sample 'network_api_pytorch_mnist' to learn the serialization process.Most the entire office code stay still,just a few lines about serialization and deserialization.I got this error:
" TypeError: runtime_deserialize_legacy() missing 1 required positional argument: 'size' "
Do you know why? @Nomow

**Comment by bobtattvamasi** [comment]
Yes, I have the same error. But recently i didn’t. So why is this happened? Maybe some dependences are changed. Did you solve your problem?

**Comment by bobtattvamasi** [comment]
I found solution for me. it's because I import tensorrt.legacy. It need just tensorrt without legacy
